compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -520,6 +524,10 @@ impl<'a> From<&'a InterpreterError> for CustomDiagnostic {
                 };
                 CustomDiagnostic::simple_error(msg, String::new(), *location)
             }
+            InterpreterError::PrefixMathError { location } => {
+                let msg = "Attempt to negate with overflow".to_string();
+                CustomDiagnostic::simple_error(msg, String::new(), *location)
+            }
```

Comment:
Can you rename this to `NegateWithOverflow`?
I'd prefer that over the somewhat ambiguous more general name, especially when the error isn't actually as general as the name makes it sound.